### ISSUE TITLE
Missing/wrong md file/directory referenced in documentation

### ISSUE THREAD

**Issue by racinmat** [JuliaRegistries/Registrator.jl#286]
`docs/src/hosting.md` states 
```
See build.md directory for instructions on how to build the docker image.
```
but I can't find this file/directory anywhere, do you mean `build.sh` file?
Thanks for clarification.

**Comment by nkottary** [comment]
The docs and the Docker stuff have not been updated in a while. After adding/cloning Registrator you can run it from the `run` directory. Or you can copy the `run` directory somewhere else on your machine.

After you fill out the `config*` files in your run folder (which used to be sample.toml) long ago, you can start Registrator with `make start-all`. You can find the full set of make targets here https://juliaregistries.github.io/Registrator.jl/dev/hosting/#Registrator-Documentation-1

**Comment by racinmat** [comment]
I see, thanks. I am trying to follow the documentation, but there is also `run.sh` file in the Dockerfile and it's not in the repo, so should I create `run.sh` file that will just call `make start-all`?

**Comment by nkottary** [comment]
Yes, you can do that. The entrypoint for docker should be blocking, otherwise the container will exit. So you need something like `tail -f *.log` at the end of your `run.sh`.
